migration/i7.1z/postgresql/202402020915_IDEMPIERE-6028.sql

```diff
@@ -0,0 +1,39 @@
+;
+
+-- 02-Feb-2024, 9:00:43 PM IST
+INSERT INTO AD_Reference (AD_Reference_ID,Name,ValidationType,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,EntityType,IsOrderByValue,AD_Reference_UU) VALUES (200265,'AD_WF_Responsible','T',0,0,'Y',TO_TIMESTAMP('2024-02-02 21:00:42','YYYY-MM-DD HH24:MI:SS'),0,TO_TIMESTAMP('2024-02-02 21:00:42','YYYY-MM-DD HH24:MI:SS'),0,'D','N','87e21ecf-cbf6-4034-b06d-2876f1c0c080')
+;
+
+-- 02-Feb-2024, 9:01:14 PM IST
+INSERT INTO AD_Ref_Table (AD_Reference_ID,AD_Table_ID,AD_Key,AD_Display,WhereClause,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,IsValueDisplayed,EntityType,AD_Ref_Table_UU) VALUES (200265,646,10483,10475,'AD_WF_Responsible.AD_Client_ID=0',0,0,'Y',TO_TIMESTAMP('2024-02-02 21:01:14','YYYY-MM-DD HH24:MI:SS'),0,TO_TIMESTAMP('2024-02-02 21:01:14','YYYY-MM-DD HH24:MI:SS'),0,'N','D','b4f0db4c-438d-43d9-bafa-316306726ded')
+;
+
+-- 02-Feb-2024, 9:01:25 PM IST
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,AD_Table_ID,ColumnName,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,SeqNo,IsEncrypted,AD_Reference_ID,AD_Reference_Value_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsSyncDatabase,IsAlwaysUpdateable,IsAutocomplete,IsAllowLogging,AD_Column_UU,IsAllowCopy,SeqNoSelection,IsToolbarButton,IsSecure,FKConstraintType,IsHtml) VALUES (216473,0,'Overridden WF Responsible',646,'Override_ID',10,'N','N','N','N','N',0,'N',30,200265,0,0,'Y',TO_TIMESTAMP('2024-02-02 21:01:24','YYYY-MM-DD HH24:MI:SS'),0,TO_TIMESTAMP('2024-02-02 21:01:24','YYYY-MM-DD HH24:MI:SS'),0,203907,'Y','N','D','N','N','N','Y','59aa0e57-6c6d-4cd0-a2df-420f9bde9783','Y',0,'N','N','N','N')
+;
+
+-- 02-Feb-2024, 9:01:29 PM IST
+UPDATE AD_Column SET FKConstraintName='Override_ADWFResponsible', FKConstraintType='N',Updated=TO_TIMESTAMP('2024-02-02 21:01:29','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_Column_ID=216473
+;
+
+-- 02-Feb-2024, 9:01:29 PM IST
+ALTER TABLE AD_WF_Responsible ADD COLUMN Override_ID NUMERIC(10) DEFAULT NULL 
+;
+
+-- 02-Feb-2024, 9:01:29 PM IST
+ALTER TABLE AD_WF_Responsible ADD CONSTRAINT Override_ADWFResponsible FOREIGN KEY (Override_ID) REFERENCES ad_wf_responsible(ad_wf_responsible_id) DEFERRABLE INITIALLY DEFERRED
+;
+
+-- 02-Feb-2024, 9:01:57 PM IST
+INSERT INTO AD_Field (AD_Field_ID,Name,AD_Tab_ID,AD_Column_ID,IsDisplayed,DisplayLength,SeqNo,IsSameLine,IsHeading,IsFieldOnly,IsEncrypted,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,IsReadOnly,IsCentrallyMaintained,EntityType,AD_Field_UU,IsDisplayedGrid,SeqNoGrid,ColumnSpan) VALUES (208378,'Overridden WF Responsible',578,216473,'Y',10,110,'N','N','N','N',0,0,'Y',TO_TIMESTAMP('2024-02-02 21:01:56','YYYY-MM-DD HH24:MI:SS'),0,TO_TIMESTAMP('2024-02-02 21:01:56','YYYY-MM-DD HH24:MI:SS'),0,'N','Y','D','6500cf64-5099-4cc8-9e2a-9b13e5842910','Y',110,2)
```

Comment:
There is a problem overriding a responsible defined as Role in System and change the type to Human in tenant.
I think is better to control that such situation must not happen.  If the System responsible is Human, then it must be a Human in tenant, same with Role.

That means probably that when filling the _Overridden WF Responsible_ field, the _Responsible Type_ must be automatically filled from system and make it read-only.

Also, is better to add a validation to ensure this in model at MWFResponsible.beforeSave